tests/DocumentationExamplesTest.php

```diff
@@ -925,6 +925,35 @@ public function testExample_55_58()
         $this->assertInventoryCount(0);
     }
 
+    public function testExample_59_61()
+    {
+        $db = new Database($this->manager, $this->getDatabaseName());
+
+        // Start Changestream Example 1
+        $cursor = $db->inventory->watch();
+        $cursor->next();
+        $next = $cursor->current();
+        // End Changestream Example 1
+
+        $this->assertNull($next);
+
+        // Start Changestream Example 2
+        $cursor = $db->inventory->watch([], ['fullDocument' => \MongoDB\Operation\ChangeStreamCommand::FULL_DOCUMENT_UPDATE_LOOKUP]);
+        $cursor->next();
+        $next = $cursor->current();
+        // End Changestream Example 2
+
+        $this->assertNull($next);
+
+        // Start Changestream Example 3
+        $resumeToken = $next->getId();
+        $cursor = $db->inventory->watch([], ['resumeAfter' => $resumeToken]);
+        $cursor->next();
+        // End Changestream Example 3
```

Comment:
I think this example should be more defensive:

```php
$resumeToken = ($current !== null) ? $current['_id'] : null;

if ($resumeToken !== null) {
    $cursor = $db->inventory->watch([], ['resumeAfter' => $resumeToken]);
    $current = $cursor->next();
}
```

It would be worth following up on [DRIVERS-414](https://jira.mongodb.org/browse/DRIVERS-414) to confirm.